packages/translate/translations.json

```diff
@@ -742,6 +742,10 @@
       "key": "api/newsletters/update/rolesNotEligible",
       "value": "Sie haben keine Berechtigung, diesen Newsletter zu abonnieren."
     },
+    {
+      "key": "api/newsletters/request/COVID19/subject",
+      "value": "Bestätigen Sie Ihre Anmeldung zum COVID-19 Newsletter."
```

Comment:
Thus far it is labeled as «Covid-19-Uhr-Newsletter». Might be worth sticking to that wording.